ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -233,27 +232,99 @@ const LocalStandardDisks = ({ idPrefix, setIsFormValid, onAddErrorNotification }
         }
     ));
 
+    const setSelectionForAllDisks = (value) => {
+        setDisks(Object.keys(disks).reduce((acc, cur) => ({ ...acc, [cur]: value }), {}));
+    };
+
+    const DropdownSplitButtonText = () => {
```

Comment:
It would be good to move this component on the top level of the file.

Reply:
Hm, I didn't think it really makes sense as a generic component outside the context of the local disks... But maybe when I actually implement the bulk logic it would make.

Reply:
Done.